plugins/available/docker.plugin.bash

```diff
@@ -49,13 +53,13 @@ function docker-image-dependencies() {
 	group 'docker'
 	if hash dot 2> /dev/null; then
 		OUT=$(mktemp -t docker-viz-XXXX.png)
-		docker images -viz | dot -Tpng > $OUT
-		case $OSTYPE in
+		docker images -viz | dot -Tpng > "$OUT"
+		case "$OSTYPE" in
```

Comment:
I'm curious if shellcheck complains about this. FWIW, `case $OSTYPE in` is harmless. The word for the `case` statement is not subject to word splitting and pathname expansions, so it's equivalent to `case "$OSTYPE" in`. The only case we want to quote the parameter expansion in the `case` statement is the array expansions of the form `${arr[*]}`, `$*`, `${!prefix*}`, etc., where Bash <= 4.4 had a bug of not respecting `IFS`.

Reply:
actually no, it didn't. I must have edited this on autopilot...